app/services/organizations/create_service.rb

```diff
@@ -10,7 +10,8 @@
 #   },
 #   organization: {
 #     name: 'Baja Pet Rescue',
-#     slug: 'baja'
+#     slug: 'baja',
```

Comment:
Updated the sample attributes to pass to the CreateService because the Org was updated to take an email as well.